boards/arc/nsim/nsim.dtsi

```diff
@@ -13,12 +13,36 @@
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
+	};
+
+	intc: arcv2-intc {
+		compatible = "snps,arcv2-intc";
+		interrupt-controller;
+		#interrupt-cells = <2>;
+	};
 
-		intc: arcv2-intc {
-			compatible = "snps,arcv2-intc";
-			interrupt-controller;
-			#interrupt-cells = <2>;
-		};
+	ici: intercore-interrupt-unit {
+		compatible = "snps,archs-ici";
+		interrupts = <19 1>;
+		interrupt-parent = <&intc>;
+	};
+
+	timer0: timer0 {
+		compatible = "snps,arc-timer";
+		interrupts = <16 1>;
+		interrupt-parent = <&intc>;
+	};
+
+	timer1: timer0 {
```

Comment:
typo, I think the cause for the issues @evgeniy-paltsev mentioned. Should be timer1: timer1.